playbooks/slack.yml

```diff
@@ -0,0 +1,28 @@
+- name: Slack
+  hosts: 127.0.0.1
+  connection: local
+  tasks:
+    - name: Check if slack is installed
```

Comment:
You can do it in one task, like below:
```
  - name: Add NordVPN Repository from deb package from Internet
    apt:
      deb: https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn-release_1.0.0_all.deb
    tags:
    - nord-vpn
    become: true
```

See more here: https://docs.ansible.com/ansible/latest/modules/apt_module.html